R/OM-methods-parsing.R

```diff
@@ -369,52 +377,31 @@ parseFOI <- function(obj, sos, verbose = FALSE) {
     
     if(verbose) cat("[parseFOI] inline FOI:", .name, "\n")
     
-    if(.name == saSamplingPointName) {
-      .sp <- parseSamplingPoint(.feature, sos = sos)
-      .foi <- GmlFeatureProperty(feature = .sp)
-    }
-    else if (.name == saSamplingSurface) {
-      warning("[parseFOI] No parsing for sa:SamplingSurface implemented!")
-      .foi <- GmlFeatureProperty(href = .name)
-    }
-    else if (.name == gmlFeatureCollectionName) {
-      .foi <- parseFeatureCollection(.feature, sos = sos)
-    }
-    else if (.name == wmlMonitoringPointName) {
-      .foi <- parseMonitoringPoint(.feature, sos = sos)
-    }
-    else {
-      warning("[parseFOI] No parsing for given feature implemented!")
-      .foi <- GmlFeatureProperty(href = .name)
-    }
-  }
-  
-  return(.foi)
-}
 
-#
-# create according GmlTimeObject from om:samplingTime
-#
-parseSamplingTime <- function(obj, format, verbose = FALSE) {
```

Comment:
Since you removed parseSamplingTime, would you mind adding a couple of tests for `parseTime(..)` ?

Reply:
On Thursday, I hope.

Reply:
https://github.com/52North/sos4R/issues/73